apps/coscrad-frontend/src/components/resources/terms/term-index.presenter.tsx

```diff
@@ -38,10 +49,21 @@ export const TermIndexPresenter = (termsIndexResult: TermIndexState) => {
             ),
         contributions: ({ contributions }: ITermViewModel) =>
             renderContributionsTextCell(contributions),
+        vocabularyLists: ({ vocabularyLists }: ITermViewModel) => (
+            <CommaSeparatedList>
+                {vocabularyLists.map(({ name }) => (
+                    <Typography>{findOriginalTextItem(name).text}</Typography>
```

Comment:
Not sure what to do here, but I'm sure I wanted to use one of the `variant`s.

Reply:
Probably `body1`